### DISCUSSION TITLE
When Fast Image/Image loading the image the onPress events are blocked

### DISCUSSION THREAD

**Discussion by sanjeevsinghofficial** [react-navigation/react-navigation#10436] [Q&A]
Hi,
I made a card consist of name and image of that person and there are 150 records and rendered in flat-list. When I press on that card then I can see that person's details on next screen.

I am facing an issue that when the image of cards are loading then onPress event is not working on that card. After image is fully loaded then all press events executed automatically which i registered during image loading.

Anyone have any solution for overcome this?

**Comment by raajnadar** [comment]
It looks more like a react native limitation, mostly this issue should go away with the new architecture.

https://reactnative.dev/architecture/fabric-renderer#motivations-and-benefits-of-the-new-renderer
